_summaries/reddi2019on_1.md

```diff
@@ -1,19 +1,53 @@
 ---
 layout: summary
```

Comment:
overall:
I think it's a little bit too short, and not all the main points are covered.
Orginaity -> good
Structure-> easy to follow

Reply:
Hi Tim,
Thank you, for your feedback. I have made the changes in the new commit. 
Since this my first review is what is the next step or who accepts the pull request?